operatorconfig/driverconfig/powerstore/v2.14.0/controller.yaml

```diff
@@ -249,12 +256,20 @@ spec:
               value: /powerstore-config/config
             - name: X_CSI_POWERSTORE_CONFIG_PARAMS_PATH
               value: /powerstore-config-params/driver-config-params.yaml
+            - name: X_CSI_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
             - name: GOPOWERSTORE_DEBUG
               value: <GOPOWERSTORE_DEBUG>
             - name: CSI_AUTO_ROUND_OFF_FILESYSTEM_SIZE
-              value: true
+              value: "true"
             - name: X_CSI_HEALTH_MONITOR_ENABLED
               value: "<X_CSI_HEALTH_MONITOR_ENABLED>"
+            - name: X_CSI_NFS_CLIENT_PORT
+              value: <X_CSI_NFS_CLIENT_PORT>
+            - name: X_CSI_NFS_SERVER_PORT
+              value: <X_CSI_NFS_SERVER_PORT>
```

Comment:
@falfaroc I had missed adding export path in my original PR but added it in https://github.com/dell/csm-docs/pull/1484. I think we need the export path in the controller. Just double check based on your testing.

Reply:
During testing, it didn't appear that we needed it but for consistency I'll add it to the controller as well 👍🏽